heal/cli/extract.py

```diff
@@ -0,0 +1,34 @@
+import click
+
+from cdislogging import get_logger
+
+from heal.vlmd.extract.extract import vlmd_extract
```

Comment:
This won't work here until the `extract` PR is merged and we pull master into this PR.